src/wp-includes/class-wp-theme-json.php

```diff
@@ -1273,8 +1281,12 @@ protected function get_css_variables( $nodes, $origins ) {
 
 			$selector = $metadata['selector'];
 
-			$node         = _wp_array_get( $this->theme_json, $metadata['path'], array() );
-			$declarations = array_merge( static::compute_preset_vars( $node, $origins ), static::compute_theme_vars( $node ) );
+			$node                    = _wp_array_get( $this->theme_json, $metadata['path'], array() );
+			$declarations            = static::compute_preset_vars( $node, $origins );
+			$theme_vars_declarations = static::compute_theme_vars( $node );
+			foreach ( $theme_vars_declarations as $declaration ) {
```

Comment:
I got a bit tripped up on these variable names. `$declaration` feels like `$declarations` is being iterated, but that's not the case here. Maybe `$theme_var_declaration` instead? Don't really feel strongly about this, it just read a bit weird to me.

Reply:
SG, will update.